### DISCUSSION TITLE
In memory of Julian and continuation of TuxGuitar

### DISCUSSION THREAD

**Discussion by depascalis** [helge17/tuxguitar#552] [General]
I've seen that you have had problems contacting the original devs of TuxGuitar, so I took it up on myself to try and contact them. I got hold of Julians brother and he explained that Julian sadly passed away two years ago. He also said that he had tried to keep the project alive but lacked some access and also didn't know anything about music. He ended with that Julians wife and kids would be really happy to hear about a continuation of Julians project in his name.

So, can we put some effort in trying to "take over" this project properly and celebrate the memory of Julian?

Since I'm not really involved in the project (yet) it would be better if you guys took over the conversation from here.
I have an email address to the brother but I'd rather share that in private. @helge17 are you on Discord or something so I can send it to you there? (I'm mostly active there nowadays)

**Comment by helge17** [comment]
I was very touched when I read your post this morning. We (@guiv42, you, me and possibly others) should definitely coordinate how we contact Julian's family and continue TuxGuitar properly.

You can reach me via email at info@tuxguitar.app, but I'll also get a Discord account in the next few days and get back to you then.

**Comment by guiv42** [comment]
Very sad news indeed.
Thanks @depascalis for taking the time to contact Julian's family. Whatever can be done in memory of Julian, I'm in.

**Comment by depascalis** [comment]
Good to hear that you are positive. Even though this is an open source project, I feel that this particular issue should be handled in a group chat due to its sensitivity, we can share the outcome later on. 

I'll initiate an email conversation, then we coordinate everybody that wants to be involved or have a say to the chat.

**Comment by belfie13** [comment]
I just read this, it's very sad.

The positive thing though, his legacy can finally be continued here.

**Comment by helge17** [comment]
Short update: We, depascalis, guiv42 and I, helge17, have tried to contact Julian's brother privately, unfortunately without success so far.

**Comment by oldvlg** [comment]
I'm late to the conversation here but just wanted you to know there is continuing interest in TuxGuitar. _Thank You_ to all who contribute in any way.
